sdk/eventhub/Azure.Messaging.EventHubs/tests/Infrastructure/EventProcessorManager.cs

```diff
@@ -260,7 +260,7 @@ public async Task WaitStabilization()
         ///   Filtering expired ownership is assumed to be responsibility of the caller.
         /// </remarks>
         ///
-        private bool AreOwnershipDistributionsTheSame(IEnumerable<PartitionOwnership> first,
+        public static bool AreOwnershipDistributionsTheSame(IEnumerable<PartitionOwnership> first,
                                                       IEnumerable<PartitionOwnership> second)
```

Comment:
Apologies the intrusion 😄 was looking amazed and fascinated by this PR and task, Great great work I was really impressed. Just noticed the indentation here.